code/modules/clothing/accessories/holster.dm

```diff
@@ -1,181 +0,0 @@
-			usr.visible_message(
-				SPAN_DANGER("[user] draws \the [holstered], ready to fight!"),
-				SPAN_WARNING("You draw \the [holstered], ready to fight!")
-				)
-		else
-			user.visible_message(
-				SPAN_NOTICE("[user] draws \the [holstered], pointing it at the ground."),
-				SPAN_NOTICE("You draw \the [holstered], pointing it at the ground.")
-				)
-		if(!user.put_in_active_hand(holstered))// If your primary hand is full, draw with your offhand
-			user.put_in_inactive_hand(holstered)// Prevents guns from getting deleted with hotkeys.
-		holstered.add_fingerprint(user)
-		playsound(user, "[sound_out]", 75, 0)
-		update_icon()
-		w_class = initial(w_class)
-		clear_holster()
-
-/obj/item/clothing/accessory/holster/attack_hand(mob/user as mob)
-	if (has_suit)	//if we are part of a suit
-		if (holstered)
-			unholster(user)
-		return
-
-	..(user)
-
-/obj/item/clothing/accessory/holster/attackby(obj/item/W as obj, mob/user as mob)
-	holster(W, user)
-
-/obj/item/clothing/accessory/holster/emp_act(severity)
-	if (holstered)
-		holstered.emp_act(severity)
-	..()
-
-/obj/item/clothing/accessory/holster/examine(mob/user)
-	..(user)
-	if (holstered)
-		to_chat(user, "\A [holstered] is holstered here.")
-	else
-		to_chat(user, "It is empty.")
-
-/obj/item/clothing/accessory/holster/on_attached(obj/item/clothing/under/S, mob/user as mob)
-	..()
-	has_suit.verbs += /obj/item/clothing/accessory/holster/verb/holster_verb
-
-/obj/item/clothing/accessory/holster/on_removed(mob/user as mob)
-	has_suit.verbs -= /obj/item/clothing/accessory/holster/verb/holster_verb
-	..()
-
-//For the holster hotkey
-/obj/item/clothing/accessory/holster/verb/holster_verb()
-	set name = "Holster"
-	set category = "Object"
-	set src in usr
-	if(!isliving(usr))
-		return
-	if(usr.stat)
-		return
-
-	//can't we just use src here?
-	var/obj/item/clothing/accessory/holster/H = null
-	if (istype(src, /obj/item/clothing/accessory/holster))
-		H = src
-	else if (istype(src, /obj/item/clothing/under))
-		var/obj/item/clothing/under/S = src
-		if (S.accessories.len)
-			H = locate() in S.accessories
-
-	if (!H)
-		to_chat(usr, SPAN_WARNING("Something is very wrong."))
-
-	if(!H.holstered)
-		var/obj/item/W = usr.get_active_hand()
-		if(!istype(W, /obj/item))
-			to_chat(usr, SPAN_WARNING("You need your weapon equipped to holster it."))
-			return
-		H.holster(W, usr)
-	else
-		H.unholster(usr)
-
-/obj/item/clothing/accessory/holster/armpit
-	name = "armpit holster"
-	desc = "A worn-out handgun holster. Perfect for concealed carry."
-	icon_state = "holster"
-
-/obj/item/clothing/accessory/holster/waist
-	name = "waist holster"
-	desc = "A handgun holster. Made of expensive leather."
-	icon_state = "holster"
-	overlay_state = "holster_low"
-
-/obj/item/clothing/accessory/holster/hip
-	name = "hip holster"
-	desc = "A handgun holster slung low on the hip, draw pardner!"
-	icon_state = "holster_hip"
-
-//Sword holsters//
-
-/obj/item/clothing/accessory/holster/saber
```

Comment:
Do not remove scabbards.

Reply:
My brother in Christ, concealed carry IS the scabbard for small blades.

Reply:
If you stand next to someone, and in the time they draw and wield a katana (both with very audible sounds) you get beaten by a "50 damage free space weapon" you deserve to die